resources/lib/NineAnimeBrowser.py

```diff
@@ -21,12 +22,24 @@ class NineAnimeBrowser(BrowserBase.BrowserBase):
     _EPISODES_RE = \
     re.compile("<li>\s<a.+?data-id=\"(.+?)\" data-base=\"(\d+)\".+?data-title=\"(.+?)\".+?href=\"\/watch\/.+?\">.+?</li>",
                re.DOTALL)
-    _EPISODE_BOXES_RE = \
-    re.compile("<i\sclass=\"fa\sfa-server\"></i>\s(.+?)\s</label>\s<div.+?>(.+?)</div>",
-               re.DOTALL)
-
     _EPISODE_LINK_RE = re.compile("<li><div><a\shref=\"/([-\w\s\d]+?)/(\d+?)\"\sclass=\"anm_det_pop\"><strong>(.+?)</strong></a><i\sclass=\"anititle\">(.+?)</i>", re.DOTALL)
 
+    settings = xbmcaddon.Addon(id='plugin.video.9anime')
```

Comment:
Do you mean I should set the settings variable (such as 'serverchoice') in default.py and then pass it to the relevant function when 'PLAY' is called?
I need that setting available as soon as the NineAnimeBrowser class is called, so it can be used in the section with the regex. To filter it later on in the code when _get_anime_info is called is something that will take me a while to figure out so would have to be put off for another weekend when I may or may not have time.